prime-router/src/main/kotlin/tokens/Jwk.kt

```diff
@@ -0,0 +1,195 @@
+package gov.cdc.prime.router.tokens
```

Comment:
After a bunch of attempts at using 3rd party libs, I ended up just writing my own JWK and JWKSet classes  See the comments in the class below, for reasons.